### PR TITLE
Updates wording and link on VaultPress notice

### PR DESCRIPTION
**Pull request by ChaosExAnima**
Fixes 1169345694087188-as-1177090507902885.

#### Changes proposed in this Pull Request:

* This updates the copy and link for the notice displayed when attempting to activate VaultPress 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of this: pbtFFM-hc-p2.

#### Does this pull request change what data or activity we track or use?
* No.

#### Testing instructions:

* On a site with Jetpack Backups purchased and the VaultPress plugin installed, activate VaultPress.

#### Proposed changelog entry for your changes:

* None needed?

Screen Shot 2020-05-22 at 1 06 57 PM

### CODE REVIEW

**Comment by folletto** [comment]
Nice. :)

I'm wondering if the copy instead of saying "new dashboard" (we don't use the dashboard label for these pages) we could just say "Jetpack.com":

Before:
> You can access your backups on your site's **Activity** page.

Current:
> You can access your backups or scan at this **new dashboard**.

Proposal: 
> You can access your backups and scan **on Jetpack.com**.

**Comment by jeffgolenski** [comment]
Good find @ChaosExAnima 

There's something about "at this new dashboard" that doesn't feel explicit enough. Do you think getting a little more specific would help? 

Or maybe I'm just crazy....well i know I'm crazy, but let's see how this feels:

>  You can access backups and scans within your new dashboard on Jetpack.com

Also, I'm not partial to it either way, but we should either use the product names "Backup & Scan" or stick with the feature names as plural "backups and scans." Whattcha think?

**Comment by ChaosExAnima** [comment]
@jeffgolenski - Given the Cloud rollout I don't want to be specific, as a good chunk of our users will be redirected to the activity log. Also Atomic users would potentially see this message!

Once the segment lib is live we can potentially do something more faceted, but until then wanted to keep it generic.

**Comment by ChaosExAnima** [comment]
@jeffgolenski - Removed reference to Scan. In the end, I decided not to be specific about the product name as this notice appears for any site with Rewind, which includes plans that have backups. I think it'd be confusing for people to mention Backup & Scan when they own neither product.

**Comment by enejb** [comment]
This change looks good to me :+1: 

Thanks for working on this!

File: 3rd-party/vaultpress.php [hunk 1]
```diff
@@ -23,7 +25,7 @@ function jetpack_vaultpress_rewind_enabled_notice() {
 				echo sprintf(
 					wp_kses(
 						/* Translators: first variable is the URL of the web site without the protocol, e.g. mysite.com */
```
Comment (enejb) [inline 3rd-party/vaultpress.php:27]:
```suggestion
						/* Translators: first variable is the full URL to the new dashboard */
```

**Comment by enejb** [comment]
Update the styles. 
Before:
Screen Shot 2020-05-25 at 10 14 37 AM

After:
Screen Shot 2020-05-25 at 10 13 41 AM

File: _inc/client/components/notice/style.scss [hunk 2]
```diff
@@ -77,14 +77,21 @@
 	}
 }
 
-.dops-notice__content {
+.dops-notice__content.dops-notice__content {
```
Comment (brbrr) [inline _inc/client/components/notice/style.scss:80]: It seems like a CSS class duplication.

Comment (enejb) [reply _inc/client/components/notice/style.scss:80]: No this is intentional to increase the specificity of the selector on the page.

Comment (enejb) [reply _inc/client/components/notice/style.scss:80]: Other wise a different selector can removed the underline of the link. Which makes the link not visible at all. 
```
.jetpack-pagestyles a {
	text-decoration: none;
}```

Comment (brbrr) [reply _inc/client/components/notice/style.scss:80]: TIL :)

File: 3rd-party/vaultpress.php [hunk 3]
```diff
@@ -22,8 +24,8 @@ function jetpack_vaultpress_rewind_enabled_notice() {
 			<?php
 				echo sprintf(
 					wp_kses(
-						/* Translators: first variable is the URL of the web site without the protocol, e.g. mysite.com */
-						__( 'You can access your backups on your site\'s <a href="https://wordpress.com/activity-log/%s" target="_blank" rel="noopener noreferrer">Activity</a> page.', 'jetpack' ),
+						/* Translators: first variable is the full URL to the new dashboard */
+						__( 'You can access your backups at this <a href="%s" target="_blank" rel="noopener noreferrer">new dashboard</a>.', 'jetpack' ),
```
Comment (dereksmart) [inline 3rd-party/vaultpress.php:28]: > new dashboard

"New" is tricky. Will this dashboard be new to first-time users? At what point will it not be new to existing users? 

Can we update this to something more future-proof?

Comment (rcoll) [reply 3rd-party/vaultpress.php:28]: > > new dashboard
> 
> "New" is tricky. Will this dashboard be new to first-time users? At what point will it not be new to existing users?
> 
> Can we update this to something more future-proof?

Agree, I'm not in love with the verbiage here. Perhaps we drop the word `new` and instead hyperlink `this dashboard`.

Comment (enejb) [inline 3rd-party/vaultpress.php:28]:
```suggestion
						__( 'You can access your backups at <a href="%s" target="_blank" rel="noopener noreferrer">this  dashboard</a>.', 'jetpack' ),
```

File: _inc/client/components/notice/style.scss [hunk 4]
```diff
@@ -77,14 +77,21 @@
...
 	@include breakpoint( ">480px" ) {
 		font-size: 14px;
 	}
+	a {
+	  text-decoration: underline;
+	  color: $white;
+	}
+  	a:hover {
+		text-decoration: none;
+	}
```
Comment (jeherve) [inline _inc/client/components/notice/style.scss:94]:
```suggestion
	a {
		text-decoration: underline;
		color: $white;
	}
	a:hover {
		text-decoration: none;
	}
```

**Pull request merged**